koiki/email.py

```diff
@@ -0,0 +1,56 @@
+import os.path
+from django.core.mail import EmailMessage
+from django.template.loader import render_to_string
+from django.utils.translation import gettext as _
+
+from koiki import wcfmmp_host, logger, error_mail_recipients
+
+
+class SuccessDeliveryMail:
+    def __init__(self, delivery):
+        self.pdf_path = delivery.print_pdf()
+        self.recipient = delivery.vendor.email
+        self.order_id = delivery.order_id
```

Comment:
These two should also be passed as arguments. Now we are tied to the email being at `delivery.vendor.email`. What if in the future we see the vendor information needs to be somewhere else and not in `delivery.vendor`? this will mean we need to change this class, potentially introducing bugs.

If you look at it, this classes the delivery object just to reach out for the things it actually cares about. It does nothing to the delivery object itself.